apps/condo/domains/miniapp/schema/B2BAppAccessRightSet.js

```diff
@@ -35,15 +75,112 @@ const B2BAppAccessRightSet = new GQLListSchema('B2BAppAccessRightSet', {
             },
         },
 
-        ...generatePermissionFields({ config: B2B_APP_SERVICE_USER_ACCESS_AVAILABLE_SCHEMAS }),
+        name: {
+            schemaDoc: 'Name for right set to distinguish it',
+            type: 'Text',
+            isRequired: true,
+            defaultValue: 'default',
+            hooks: {
+                validateInput ({ resolvedData, existingItem, context }) {
+                    const nextItem = {
+                        ...(existingItem || {}),
```

Comment:
```suggestion
                        ...existingItem,
```

Spreading undefined is safe